### ISSUE TITLE
Set up README

### ISSUE THREAD

**Issue by hugoormo** [sonofmbse/SysML-v2#6]
# Standard Workflow:
## Concept of Work:
The source data is located at "System-Modeling/SysML-v2-Release". We have no push rights to this repository.
We have forked that source into our public repository at "sonofmbse/SysML-v2". We have push rights to this repository.
Our model is stored at a folder in our fork called ...sysml/src/EBike.
We create new features locally in our computers at a feature-branch.
We push new features to a feature-branch at the fork.
We issue pull-requests (PR) from the feature-branch at the fork into the master at the fork.
We pull the newest source-data from the source centrally from the fork. Each collaborator pulls the newest source-data from the fork.

## Workflow
1. Clone the fork-repo in your PC.
2. Open an issue in the fork with the description of the feature you want to work on. 
3. Once the issue has been agreed upon the asignee will set up a "feature branch"
4. The assignee make sure that she has the latest status of the master localy. She makes a pull. 
5. The assignee works on the new feature on her branch and pushes the new content to the feature branch in GitHub.
7. Once the content is ready to be merged into the master branch the contributor shall generate a pull request from the feature branch to the master branch. Pay attention to generate the pull request to the master branch of the fork (not of the source data repository at "System-Modelling/SysML-v2".
8. The contributor has to keep the pull request free of conflicts in order to ease the review and merge of the pull request. 
9. The working team will review weekly the pull requests and merge them into the master.

**Comment by martinbu1** [comment]
### Concept of Work

![Concept_of_Work](https://github.com/sonofmbse/SysML-v2/assets/150663841/b348a17a-5a09-45c2-899f-b9bd6695a1f1)

**Comment by kirchner95** [comment]
Kannst du die Präsentation verlinken bzw. ablegen, das man sie anpassen kann. Feature-Fork müsste Feature Branch sein glaube ich.
